app/models/user.rb

```diff
@@ -67,6 +67,15 @@ def rating_record
     client.find_rating_info(participant_id).body.fetch('ratingRecord', {})
   end
 
+  # This is a helper method for pulling mhv_correlation_id
+  def mhv_correlation_id
+    @mhv_correlation_id ||= begin
+      return nil unless mvi.present?
+      return nil unless mvi[:mhv_id]
```

Comment:
@aub i think that's desired behavior.

I think this line can be deleted in favor of the safe navigator:
```ruby
mvi[:mhv_id]&.split('^')&.first
```